src/Symfony/Component/Notifier/Bridge/Bluesky/Tests/BlueskyTransportTest.php

```diff
@@ -264,6 +274,41 @@ public function testParseFacetsUrlWithTrickyRegex()
         $this->assertEquals($expected, $this->parseFacets($input));
     }
 
+    public function testWithMedia()
+    {
+        $transport = $this->createTransport(new MockHttpClient((function () {
+            yield function (string $method, string $url, array $options) {
+                $this->assertSame('POST', $method);
+                $this->assertSame('https://bsky.social/xrpc/com.atproto.server.createSession', $url);
+
+                return new MockResponse('{"accessJwt":"foo"}');
```

Comment:
You could leverage new JsonMockResponse and provide the response as PHP, same for the others

```suggestion
                return new JsonMockResponse(['accessJwt' => 'foo']);
```